### PR TITLE
Allow setting a custom terminal title

### LINKED ISSUE #62: Configurable author & terminal
Would you add (or accept a PR to add) customization of the author passed to `git-standup`, and the ability to turn off the terminal title setting?

Thanks for this great software!

### PR DESCRIPTION
**Pull request by aripollak**
Maybe it's a bit overkill to allow setting an arbitrary title, but here it is! :tada: 

Closes #62.

### CODE REVIEW

**Comment by notwaldorf** [comment]
This looks great, but needs a rebase! :)

**Comment by aripollak** [comment]
@notwaldorf fixed!

File: config.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ var config = {
   // Set to false if you're an imperial savage. <3
   celsius: (process.env.TTC_CELSIUS || 'true') === 'true',
 
+  terminal_title: (process.env.TTC_TERMINAL_TITLE === 'false' ? null : '✨💖 tiny care terminal 💖✨'),
```
Comment (notwaldorf) [inline config.js]: This won't actually set it to any other string like in the instructions

Comment (aripollak) [inline config.js]: Whoops. Fixed the instructions so only disabling is supported.

**Comment by notwaldorf** [comment]
Thanks!

**Pull request merged**